lib/engine/game/g_rolling_stock/entities.rb

```diff
@@ -440,7 +440,7 @@ module Entities
             text_color: 'white',
             tokens: [],
             shares: [10, 10, 10, 10, 10, 10],
-            float_percent: 10,
+            float_percent: 50,
```

Comment:
what's up with float percent?

Reply:
It's the number that causes it to float correctly because the stock in RS Stars doesn't add up to 100%